app/views/timelog/_timelog_edit_bottom.erb

```diff
@@ -2,6 +2,10 @@
   <em><%= l(:label_no_data) %></em>
 <% else %>
   <% user_options = time_entry.project.users.map {|p| [p.name, p.id]} %>
-  <% selected_user = time_entry.user.id || User.current.id %>
+  <% if !time_entry.user.nil? %>
```

Comment:
Won't these evaluate to the same thing?

Reply:
The way it was written before it assumed there were always a user object in time_entry. It looks like it's not the case anymore so this was failing